### PR TITLE
Update guide.rst

### PR DESCRIPTION
**Pull request by radogit**
added a few links to slides
added a few presentations
corrected a date

### CODE REVIEW

File: content/pages/guide.rst [hunk 1]
```diff
@@ -249,6 +249,15 @@ Past Meeting Topics and Materials
      - Date
      - Presenters
      - Materials
+   * -
```
Comment (moorepants) [inline content/pages/guide.rst:252]:
```suggestion
   * - Learning skateboarding with feedback used in sports
```

File: content/pages/guide.rst [hunk 2]
```diff
@@ -266,24 +275,33 @@ Past Meeting Topics and Materials
      - 2021-10-26
      - Jan Heinen
      - 
-   * - 
+   * - Steer-assist Bicycle
      - 2021-10-12
      - Leila Alizadehsaravi 
-     - 
+     - `Slides 
+     <https://drive.google.com/file/d/1V-NL3dBGg8gWIR3ibsKjyFQqwSg2TBvx/view?usp=sharing>`__
```
Comment (moorepants) [inline content/pages/guide.rst:282]: I'm not sure everyone wanted to share these publicly. We need to check with people before adding the link here.

Comment (radogit) [reply content/pages/guide.rst:282]: This (2021-10-12) presentation link can be deleted.

File: content/pages/guide.rst (hunk 1 above)
Comment (radogit) [reply content/pages/guide.rst:252]: sounds great.

**Comment by radogit** [comment]
sent out emails asking for making slides public to Marco, Julie, Leila, Tim, Jan H, Jelle.

Got response from 
- Leila (forwarded, implemented: 3 slidedecks cleared, 1 removed), 
- Jan H (Tim has the files), 
- Jelle (confirmed by Element and cleared, awaiting email confirmation for archiving sake).

**Comment by radogit** [comment]
Jelle (cleared, email forwarded)

**Comment by moorepants** [comment]
I don't think Leila can share these things about the steer assist bike
publicly. She has to get permission from Gazelle and Bosch first.

Jason

**Comment by moorepants** [comment]
Closing as web page has diverted. Reopen if you want to make these fixes.

**Pull request closed without merging**